### PR TITLE
6 get/api/articles/article id/comments

### PR DESCRIPTION
**Pull request by Phil-Code**

### CODE REVIEW

File: __tests__/endpoints.test.js [hunk 1]
```diff
@@ -119,4 +118,57 @@ describe('GET/api/articles/:article_id', ()=>{
...
+            const sampleComment = {
+                comment_id: 11,
+                body: "Ambidextrous marsupial",
+                votes: 0,
+                author: "icellusedkars",
+                article_id: 3,
+                created_at: "2020-09-19T23:10:00.000Z",
+              }
+            expect(comments[0]).toMatchObject(sampleComment)
+        })
```
Comment (NRMSMITH) [inline __tests__/endpoints.test.js:141]: I think you should probably check that all the comments match the object -> so you need to have a look at a way of dynamically checking them all :) (a length check and then a forEach is probably the nicest way of doing it!)

File: __tests__/endpoints.test.js [hunk 2]
```diff
@@ -119,4 +118,57 @@ describe('GET/api/articles/:article_id', ()=>{
...
+    })
+    test('comments are sorted in date order, the most recent first', ()=>{
+        return request(app)
+        .get('/api/articles/3/comments')
+        .then(({body})=>{
+            const {comments} = body
+            expect(comments).toBeSortedBy('created_at', {descending: true})
+        })
```
Comment (NRMSMITH) [inline __tests__/endpoints.test.js:149]: 👍

File: __tests__/endpoints.test.js [hunk 3]
```diff
@@ -119,4 +118,57 @@ describe('GET/api/articles/:article_id', ()=>{
...
+            expect(body.msg).toBe('resource not found')
+        })
+    })
+    test('sends 400 status code and "bad request" message when client requests an article with an invalid id type', ()=>{
+        return request(app)
+        .get('/api/articles/invalid_request/comments')
+        .expect(400)
+        .then(({body})=>{
+            expect(body.msg).toBe('bad request')
+        })
```
Comment (NRMSMITH) [inline __tests__/endpoints.test.js:174]: 👍

**Review by NRMSMITH — CHANGES_REQUESTED** [review]
Rewrite your first test for this endpoint - please resubmit once it's done 🚀

**Review by pippaamy — APPROVED** [review]
great changes, go ahead and merge :)

**Pull request merged**